Added threshold lines for the NovaSeqs in the "chemistries"-view and changed the colors

sofiahaglund opened the pull request:
I added plot lines for the threshold values (what is a pass) for the NovaSeqs (only for clusters and flowcell views. Since the colors made it slightly difficult to differentiate between the different flowcells, I also changed the colors to unicorn colors! <3

alneberg commented:
Looks good! I'm not the one to give graphical design advices, but how would it look with width 1 instead of 2 for the dotted lines?

For me it would make sense to make this the default first view of this page as well. What do you think?

sofiahaglund commented:
> Looks good! I'm not the one to give graphical design advices, but how would it look with width 1 instead of 2 for the dotted lines?
> 
> For me it would make sense to make this the default first view of this page as well. What do you think?

I made the lines narrower, it's up on stage now, you're right, maybe it looks better?

I agree, it would make sense to make it the default view. What do other people think?

I think I need to also simplify the code specifying the colors, I just sort of played with it until I got the colors I wanted. But I don't think the syntax is entirely correct :-)

sofiahaglund commented:
I made the line colors less saturated, fixed the S4 line (that was hidden behind the grid line), and changed the default view to the "color by chemistry, display clusters/flowcells."
It's up on stage. What do you think?
